benchmark-e2e/benchmark-e2e.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+  id 'java'
+  id "io.freefair.lombok" version "5.1.0"
```

Comment:
We don't use lombok, can you remove this? It looks like you aren't using it either